base/applications/mspaint/winproc.h

```diff
@@ -28,7 +28,16 @@ class CMainWindow : public CWindowImpl<CMainWindow>
         MESSAGE_HANDLER(WM_MOUSEWHEEL, OnMouseWheel)
     END_MSG_MAP()
 
+    CMainWindow() : m_hMenu(NULL) { }
+
+    HWND DoCreate();
+    BOOL DoGetOpenFileName(IN OUT LPTSTR pszFile, INT cchMaxFile);
+    BOOL DoGetSaveFileName(IN OUT LPTSTR pszFile, INT cchMaxFile);
+    BOOL DoChooseColor(IN OUT COLORREF *prgbColor);
```

Comment:
I'm not convinced by the need to add "Do" to all those function names.

Reply:
Reduced 'Do's.